skimage/transform/hough_transform.py

```diff
@@ -17,15 +17,15 @@ def hough_line(img, theta=None):
         Input image with nonzero values representing edges.
     theta : 1D ndarray of double
         Angles at which to compute the transform, in radians.
-        Defaults to -pi/2 .. pi/2
+        Defaults to -pi/2 .. pi/2 with 180 values.
 
     Returns
     -------
-    H : 2-D ndarray of uint64
+    hspace : 2-D ndarray of uint64
```

Comment:
Considering the actual variable names here don't matter to the user in any way, I'm fine with being more descriptive and following capitalization conventions. `distances` might be better than `dists` though.